mesh/mesh_readers.cpp

```diff
@@ -2681,9 +2684,9 @@ void Mesh::ReadGmshMesh(std::istream &input, int &curved, int &read_gf)
          // Upon completion of this loop, each v2v[slave] will point to a true
          // master vertex. This algorithm is useful for periodicity defined in
          // multiple directions.
-         for (int slave = 0; slave < v2v.Size(); slave++)
+         for (slave = 0; slave < v2v.Size(); slave++)
```

Comment:
I would prefer `slave` and `master` to be declared in a narrower scope.

Reply:
Done